lib/site_prism/waiter.rb

```diff
@@ -14,5 +14,13 @@ def self.wait_until_true(wait_time = Capybara.default_max_wait_time)
 
       raise SitePrism::TimeoutError, "Timed out after #{wait_time}s."
     end
+
+    def self.wait_until_displayed(*args, url)
+      args.last.is_a?(::Hash) ? args.pop : {}
+      seconds = !args.empty? ? args.first : Capybara.default_max_wait_time
+      raise SitePrism::NoUrlMatcherForPageError unless url
```

Comment:
Lets optimise this method call by making this a proper guard clause on line1 of the call